src/main/java/com/endava/soa_5/basic/Exercise_2.java

```diff
@@ -0,0 +1,32 @@
+package com.endava.soa_5.basic;
+
+public class Exercise_2 {
+    //Write a method that prints if a number is divisible by 3 or 5.
+
+    int number;
+
+    Exercise_2() {
+    }
+
+    Exercise_2(int number) {
+        this.number = number;
+    }
+
+    String setNumberForResult() {
+        String message;
+        if (number % 3 == 0 && number % 5 == 0) {
+            message = number + " \"Is divisible by 3 and 5\"";
+
+        } else if (number % 3 == 0) {
+            message = number + " \"Is divisible by 3\"";
+
+        } else if (number % 5 == 0) {
+            message = number + " \"Is divisible by 5\"";
+
+        } else {
+            message = number + "\"Isn't divisible by 3 and 5\"";
```

Comment:
add a space in the string before the actual message, like in the messages above

Reply:
Done!